Marlin/src/lcd/language/language_en.h

```diff
@@ -543,8 +543,8 @@
 #ifndef MSG_HOTEND_TOO_COLD
   #define MSG_HOTEND_TOO_COLD                 _UxGT("Hotend too cold")
 #endif
-#ifndef MSG_MOVE_0025MM
-  #define MSG_MOVE_0025MM                     _UxGT("Move 0.025mm")
+#ifndef MSG_MOVE_001MM
+  #define MSG_MOVE_001MM                      _UxGT("Move 0.01mm")
```

Comment:
There are 28 more language files that will require the same change.

Reply:
i tried with 0.01 first... but even with a Z driver which do 400 steps per mm... 5/10 steps is almost a limit to be accurate, avoid Z axis grip, etc. I mean it will not work well for 200/mm ones... 

Here with 400, 3 steps do nothing on Z

Reply:
also, quarters in circles (TR8 axis) are easier to mesure

Reply:
Well, 400 Steps and TMC5160 with 64 micosteps with a 20 tooth GT2 pulley is about 740 steps per mm which is about 0.00135 mm per step, and I can tell you, there is actually is movement! But on a delta movement is translated, which means that some steps are larger, and some are smaller. There is no fixed accurcy value!
So either we keep metric steps of 1/10th or we add a custom step value...

Reply:
GT2 pulleys on Z ?

Reply:
> GT2 pulleys on Z ?

Delta printer does not care about X Y Z... all the same accuracy.

Reply:
I suppose it will have to be an option. But now I am curious. What kind of work do you do that requires under 1/40mm accuracy? Note that if you're using one of the mesh bed leveling systems that kind of accuracy is out the window because it's all approximation and the inaccuracies between the measured mesh points commonly exceed 0.05mm.

Reply:
Remember:
The minimum distance for a move is MIN_STEPS_PER_SEGMENT.
Moving n steps forward needs going MIN_STEPS_PER_SEGMENT steps back, than MIN_STEPS_PER_SEGMENT +n forward, when n < MIN_STEPS_PER_SEGMENT.
Or Babysteping.

For normal DELTAS with 80 steps/mm = 0.0125mm/step and 0.025mm moves this is already relevant.

Reply:
> The minimum distance for a move is MIN_STEPS_PER_SEGMENT.
I did not know that! If that really is true, even for manual moves, my accuracy would be 0,0081, which explains why my 0.01 moves still worked in my case. But stepping back, and than forward as suggested seems a little bit ugly, but as long as it works... and it would only be needed if movement is actually smaller than MIN_STEPS_PER_SEGMENT. (That calculations could be done by precompiler, so there should be no extra memory needed when this logic is not needed)
I wonder if babystepping has a callable function for a behaviour like that ?!

Reply:
> What kind of work do you do that requires under 1/40mm accuracy?
Manual bed leveling and delta calibration. I can hear my nozzle touching the bed (the fan vibrations), and with 0.1mm layer height 0.01mm still is as much as 10% of a layers height! I am using a 0.2mm nozzle, and would like to go even smaller some day... ;-) 
And of course there is PCB milling where leveling is crucial, too!

Reply:
> And of course there is PCB milling where leveling is crucial, too!

Assuming your PCBs and material workpieces are flat, tramming is going to be far more crucial. You want the mechanism itself to be as true as possible.